src/doc/en/prep/Quickstarts/Linear-Algebra.rst

```diff
@@ -313,12 +313,8 @@ eigenvectors.
 ::
 
     sage: D,P=H.eigenmatrix_right()
-    sage: P*D-H*P
-    [0 0 0 0 0]
-    [0 0 0 0 0]
-    [0 0 0 0 0]
-    [0 0 0 0 0]
-    [0 0 0 0 0]
+    sage: P*D-H*P == matrix.zero(5)
+    True
```

Comment:
This is necessary because now `check=False` is passed, so `exactify` is no longer called, so it shows some `0.000000?e-16` thing instead.